distributed/tests/test_spill.py

```diff
@@ -1,16 +1,18 @@
 from __future__ import annotations
 
+import array
 import logging
 import os
+import random
 import uuid
 
+import py
```

Comment:
I find it really hard to correlate this module to pytest (because pytest can also be imported as py.test). I think it would be just simpler to declare those parameters as Any.

Reply:
Makes sense!

Reply:
Migrated to the recommended `tmp_path` and `tmp_path_factory` and therefore to `pathlib.Path`, which should solve this entire issue.